fb-install-puppet.bats

```diff
@@ -66,17 +66,17 @@ load os_helper
 @test "setup Puppet Labs nightly repos" {
   [ x${PUPPET_REPO} = xnightly ] || skip "PUPPET_REPO is not set to nightly"
   tSetOSVersion
+  tPackageExists curl || tPackageInstall curl
   if tIsFedora; then
-    tPackageExists puppet5-nightly-release || \
-      rpm -ivh http://yum.puppetlabs.com/puppet5-nightly/puppet5-nightly-release-fedora-${OS_VERSION}.noarch.rpm
+    curl -o /etc/yum.repos.d/puppet-agent.repo http://nightlies.puppetlabs.com/puppet-agent-latest/repo_configs/rpm/pl-puppet-agent-latest-fedora-${OS_VERSION}-x86_64.repo
```

Comment:
How about using ```wget -O``` like we do in the rest of the file?

Reply:
good point, changed.